tests/test_s3parallelcopy.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+import sys
+import unittest
+import uuid
+from concurrent.futures import ThreadPoolExecutor
+
+import boto3
+import botocore
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from dss import stepfunctions
+from dss.stepfunctions import s3copyclient
+from dss.stepfunctions.s3copyclient.implementation import LAMBDA_PARALLELIZATION_FACTOR
+from dss.util.aws import AWS_MIN_CHUNK_SIZE
+from tests import eventually, infra
+from tests.infra import testmode
+
+
+class TestS3ParallelCopy(unittest.TestCase):
+    @staticmethod
```

Comment:
Why is this static?

Reply:
It has no dependency on the class or the instance.  It is merely here for code organizational purposes.

Reply:
I used to religiously mark every method that didn't reference `self` as static but stopped doing that because it turned out to be incidental. Subsequent revisions of the code suddenly introduced references to `self` and the decoration had to be removed. This got especially annoying if the method is called through the class, like you do below.

As far as code organization is concerned, it would send a stronger message to there reader if this method were protected.